### DISCUSSION TITLE
Logout

### DISCUSSION THREAD

**Discussion by szymonlisiecki** [Glench/ExtPay#93] [General]
Is there a way to logout a user other than removing extension? If not - are you planning to implement logout?

**Comment by burkybang** [comment]
What's your use case? If the user has stopped paying, their license will become inactive.

**Reply by szymonlisiecki** [reply]
We have the "login" functionality - naturally would be to also have the logout functionality. The thing is that you can actually log out - you have to remove the extension and install it again. So this process could be shortened by the "log out" button.

I don't have a special case - I thought about it while testing the app.

**Reply by burkybang** [reply]
Logging out is easy. Remove the sync storage that gets saved when you log in.

I thought you were asking for a way to remotely log someone out.

**Reply by szymonlisiecki** [reply]
Oh, thanks, mate. Indeed it works. Cheers

**Reply by burkybang** [reply]
Happy to help!

**Reply by Glench** [reply]
Yes thanks for your help!
